sig-and-wg/sig-core/proposals/installer.md

```diff
@@ -0,0 +1,44 @@
+# Kyma Installer - dynamic installation 
+
+Created on 2018-08-23 by Krysian Cieślik (@crabtree) and Jakub Kabza (@jakkab).
+This document is a part of the [Kyma modularization proposal](./modularization.md).
+
+## Status
+
+Proposed on 2018-08-23.
+
+## Motivation
+
+Kyma installation is a complex, time-consuming process, involving multiple components. The procedure is followed every day by a large number of developers who test their features, and thus should easily handle different scenarios without modifications. This is where the dynamic installation comes in handy, unifying, simplifying and automating the process.
+
+## Goal
+
+This proposal aims to enhance developer experience by:
+
+- extending installation customizability 
+- reducing the number of different installation methods
+- avoiding redundant modifications to the Installer component
+
+## Current solution
+
+Kyma installation:
+
+- consists of hard-coded steps
+- uses a variety of installation methods throughout the process, such as custom shell scripts, Helm, and kubectl commands
+- stores overrides in a fixed struct that requires code modifications every time new parameter is introduced
+
+All things considered, the installation process lacks coherence and leaves little room for customization, requiring constant maintenance at the same time.
+
+## Proposal
+
+Suggested modifications:
+
+- Installer no longer supports Bash scripts.
+- In order to increase security, Docker container used to run Installer does not feature Bash at all.
+- Available installation and update methods are limited to Helm and kubectl commands.
```

Comment:
what does it mean?

Reply:
@jakkab  this sentence should be rephrase.

Reply:
rephrased.